00 - Notes/Scalability.md

```diff
@@ -0,0 +1,22 @@
+---
+version: "0.0.0"
+published: true
+---
+uplinks:: [[Blockchain Trilemma คือ]]
+tags::#type/note #lang/th #people/Sutida
+
+# Scalability
```

Comment:
Scalability ของ Blockchain ใช่ไหมครับ ถ้าใช่ควรแก้หัวข้อเป็น Scalability of Blockchain